converters/obs_converter.py

```diff
@@ -25,9 +36,8 @@ def __init__(self, lang_code, git_repo, out_dir, quiet):
         self.git_repo = git_repo
         self.out_dir = out_dir
         self.quiet = quiet
-        # self.temp_dir = ''
 
-        if 'github' not in git_repo and 'file://' not in git_repo:
+        if 'github' not in git_repo and 'file://' not in git_repo and 'git.door43' not in git_repo:
```

Comment:
Add dokuwiki repos are on github. What is the purpose of including 'git.door43' here?